src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,55 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        writeToFile(createReport(fromFileName), toFileName);
+    }
+
+    private String readFromFile(String fileName) {
+        try {
+            return Files.readString(Path.of(fileName));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read from file " + fileName, e);
+        }
+    }
+
+    private String createReport(String dataFromFile) {
+        final String supply = "supply";
+        final String buy = "buy";
+        int supplySum = 0;
+        int buySum = 0;
+
+        String[] arrayFromFile = readFromFile(dataFromFile).split("\\W+");
```

Comment:
```suggestion
        String[] arrayFromFile = dataFromFile.split("\\W+");
```
could also extract regex to a constant

Reply:
ok, done